src/gluonts/shell/serve.py

```diff
@@ -121,7 +122,8 @@ def handle_error(error) -> Tuple[str, int]:
 
     @app.route("/ping")
     def ping() -> Response:
-        app.logger.info("Responding to /ping request")
+        time = datetime.now().isoformat()
+        app.logger.info(f"Responding to /ping request at {time}")
```

Comment:
Couldn't we just adapt the logger to also print the time?

Reply:
Done